lib/collection/src/operations/verification/update.rs

```diff
@@ -239,6 +240,24 @@ async fn check_collection_size_limit(
         check_collection_payload_size_limit(payload_storage_size_limit_bytes, stats)?;
     }
 
+    if let Some(points_count_limit) = point_limit {
```

Comment:
Would it make sense to include the number of points the current request might add? As far as I can see we're not doing that.

Imagine we set a point limit of 1. It looks like the first request is able to insert a 100 points in one operation, and only then we're limiting point count. Or am I wrong on that?

Reply:
You are right, it would be better to not assume each request is equivalent to adding one point.
I will see what  I can do here :+1:

Reply:
@agourlay I'd be fine with handling that in a separate PR, so we keep this one simple and clean.

I don't deem this super critical, but rather a nice to have.